raven/README.md

```diff
@@ -42,6 +42,14 @@ Multiple values of the permission granting service resourceID(s) may be provided
 settings. All services will be consulted but a single positive result is sufficient to grant permission for an 
 action.
 
+Another set of optional keys configure raven to prioritize sites returned in a transfer, with higher priority
+sites first in the list of transfer URL's to GET or PUT a file.
+```
+org.opencadc.raven.putPreference={storage site name}
+{storage site name).resourceID={storage site resourceID}
+{storage site name).namespaces={comma delimited list of namespaces}
```

Comment:
Don't use comma-delimited - the config supports multi-valued keys and you can get a List<String> without more parsing.

Also, `namespace` singular